src/main/java/org/apache/commons/io/input/Tailer.java

```diff
@@ -191,6 +203,18 @@ public Tailer(final File file, final TailerListener listener, final long delayMi
         this(file, listener, delayMillis, false);
     }
 
+    /**
```

Comment:
There are way too many new factory APIs here IMO. For the first cut, let's add one and see what other use-cases require in the future. If a delay is provided, please make that a `Duration`, not a `long`. I'll redo the internals after this PR comes in to use a Duration instead of a `long`. If you add a constructor, make it private or package private.